doc/changelog.rst

```diff
@@ -1,6 +1,61 @@
 Changelog
 =========
 
+0.3.0
+-----
+
+Expected Release on Febuary 11, 2022
+
+Description
+
+ - Update Redis version to 1.3.2 (PR162_)
+ - Streamline ``run`` and ``connect`` commands for Redis and RedisCluster (PR203_)
+ - Update handling of command replies (PR138_) (PR163_)
+ - Add SmartRedis namespace to ``srassert.h`` (PR191_)
+ - Expose Redis commands to the Python client (PR139_) (PR160_)
```

Comment:
Maybe we can list out the functions: FLUSHDB, CONFIG GET, CONFIG SET, SAVE

Reply:
Probably wise to say _what_ functionality was exposed. Fixed in https://github.com/CrayLabs/SmartRedis/pull/209/commits/a8561c47fe1c2d31271f36ac01b711311ae7d028